### PR TITLE
[TASK] Rename API end point

### PR DESCRIPTION
**Pull request by gilbertsoft**

### CODE REVIEW

File: config/packages/nelmio_api_doc.yaml [hunk 1]
```diff
@@ -9,4 +9,4 @@ nelmio_api_doc:
                     type: basic
     areas: # to filter documented areas
         path_patterns:
-            - ^/v1/api(?!/doc$) # Accepts routes under /api except /api/doc
+            - ^/api/v1(?!/doc$) # Accepts routes under /api except /api/doc
```
Comment (Crell) [inline config/packages/nelmio_api_doc.yaml:12]: This is a significant API change, no?

File: src/Controller/Api/CacheController.php [hunk 2]
```diff
@@ -35,10 +35,8 @@
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
-/**
- * @Route("/v1/api/cache", methods={"DELETE"})
- * @IsGranted("ROLE_ADMIN")
- */
+#[Route('/{api_entry_point}/cache', requirements: ['api_entry_point' => 'api/v1|v1/api'], defaults: ['_format' => 'json'])]
```
Comment (Crell) [inline src/Controller/Api/CacheController.php]:
```suggestion
#[Route('/{api_entry_point}/cache', 
    requirements: ['api_entry_point' => 'api/v1|v1/api'], 
    defaults: ['_format' => 'json']),
]
```

Or possibly name even the first argument, for clarity/consistency.

File: src/Controller/Api/MajorVersion/ReleasesController.php [hunk 3]
```diff
@@ -38,9 +38,7 @@
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 use Symfony\Component\Routing\Annotation\Route;
 
-/**
- * @Route("/v1/api/major/{version}/release", defaults={"_format": "json"})
- */
+#[Route('/{api_entry_point}/major/{version}/release', requirements: ['api_entry_point' => 'api/v1|v1/api'], defaults: ['_format' => 'json'])]
```
Comment (Crell) [inline src/Controller/Api/MajorVersion/ReleasesController.php]: Ibid.

File: config/packages/nelmio_api_doc.yaml (hunk 1 above)
Comment (gilbertsoft) [reply config/packages/nelmio_api_doc.yaml:12]: Yes, but that's the api docs only. For the real api entry points there is a fallback to the existing url see bellow.

File: src/Controller/Api/CacheController.php (hunk 2 above)
Comment (gilbertsoft) [reply src/Controller/Api/CacheController.php]: I've had that too but currently my IDE does not like multiliners. Have to get rid of it first...

**Pull request merged**